exp/views/study.py

```diff
@@ -1326,6 +1318,73 @@ def dispatch(self, request, path, *args, **kwargs):
         return super().dispatch(request, path)
 
 
+def get_flattened_responses(response_qs, studies_for_child):
+    """Get derived attributes for children.
+
+    TODO: consider whether or not this work should be extracted out into a dataframe.
+    """
+    response_data = []
+    for resp in response_qs:
+        child_age_in_days = (datetime.date.today() - resp.child.birthday).days
+        languages_spoken = popcnt_bitfield(
+            int(resp.child.languages_spoken), "languages"
+        )
+        response_data.append(
+            {
+                "Response (unique identifier)": resp.uuid,
+                "Child (unique identifier)": resp.child.uuid,
+                "Child Age in Days": child_age_in_days,
+                "Child Age in Months": round(child_age_in_days / 30, 2),
```

Comment:
rounding to two decimal places gives approx. same precision as days - can just do int(child_age_in_days / 30) to get rounded down to nearest integer. (I think that'll be more straightforward to interpret than actual rounding, esp. for years.)